ulib/libax/src/cbindings/stat.rs

```diff
@@ -0,0 +1,15 @@
+use super::ctypes::mode_t;
+use crate::debug;
+
+static mut MASK: mode_t = 0o666;
+
+/// Set umask for open operations
+///
+/// Currenly only a fake implementation
+#[no_mangle]
+pub unsafe extern "C" fn ax_umask(mask: mode_t) -> mode_t {
```

Comment:
Move this fake implementation to c_libax?

Reply:
remove this implementation